app/models/application_choice.rb

```diff
@@ -78,6 +78,10 @@ def days_until_decline_by_default
     end
   end
 
+  def offer
```

Comment:
What was the issue with defining the association for `offer`?

Reply:
It was related to test data creation. Adding the offer association to the `with_offer` scope of the application choice factory was causing it to generate multiple application choices which as a result caused irrelevant specs who relied on a specific number of applciation choices being generated to fail, so I rolled it all back and decided to just leave it out for now.

Reply:
Will we add this back in the refactor card then? It's not too important since this does the same thing anyway

Reply:
I don't know if it should be done as part of the refactor card. It may be simpler to wait until the offer field has been removed first. Maybe a small tech debt card.